src/scrapers/hapoalim-business.js

```diff
@@ -0,0 +1,93 @@
+import puppeteer from 'puppeteer';
+import fs from 'fs';
+import {
+  waitUntilElementFound,
+  clickButton,
+} from '../helpers/elements-interactions';
+import { askQuestion } from '../helpers/waiting';
+import { waitForRedirect } from '../helpers/navigation';
+import { addMonths } from '../helpers/dates';
+import { fetchAccountData } from './poalim-business/helpers';
+import { SCRAPE_PROGRESS_TYPES } from '../constants';
+import { BaseScraper } from './base-scraper';
+
+const VIEWPORT_WIDTH = 1024;
+const VIEWPORT_HEIGHT = 768;
+
+const BASE_URL = 'https://biz2.bankhapoalim.co.il/authenticate/logon/main';
+
+class HapoalimBusinessScraper extends BaseScraper {
```

Comment:
this works.  but I'm sure there are functions here that I can reuse from other scrapers.
I wanted to get your ideas about it because I also like to have a bare bone thing that is easier to debug